cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -46,6 +46,11 @@ class PythonDocCommentFormatter final : public DocCommentFormatter
     {
         return Slice::Python::pyLinkFormatter(rawLink, source, target);
     }
+
+    string formatSeeAlso(const string& rawLink, const ContainedPtr& source, const SyntaxTreeBasePtr& target) final
+    {
+        return Slice::Python::pyLinkFormatter(rawLink, source, target);
+    }
```

Comment:
This isn't necessary, `formatSeeAlso` is default-implemented to call `formatLink`:
https://github.com/zeroc-ice/ice/blob/daaa6e39bde259b7f8089c60db81d18eb29c7eff/cpp/src/Slice/DocCommentParser.cpp#L68-L73
and indeed, this has the same implementation as `formatLink`.